notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>Level 2 header here too</p><p> </p><p><code>## Scrubbers!</code></p><p> </p><p>Change <code>[!NOTE]</code> to <code>!TIP]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>